cli/features/abbreviations.feature

```diff
@@ -5,13 +5,13 @@ Feature: executing abbreviated Exosphere commands
   I want to be able to use abbreviations for commands
   So that I can use Exosphere without having to type so much.
 
-  Scenario Outline: executing "exo add" with abbreviated commands
-    When executing the abbreviated command "<ADD-COMMAND>" in the terminal
-    Then the full command "exo add" is executed
+  Scenario Outline: executing "exo add-template" with abbreviated commands
```

Comment:
We can omit these abbreviated commands if it helps keep things simpler. The idea didn't really hold up so far and could be added later.